src/guide/transitions-list.md

```diff
@@ -300,7 +300,7 @@ Vue.component('my-special-transition', {
 })
 ```
 
-And [functional components](render-function.html#Functional-Components) are especially well-suited to this task:
+And [functional components](render-function.html#functional-components) are especially well-suited to this task:
```

Comment:
This section needs major work to update it for Vue 3 but at least this link goes somewhere now.